hit-test.html

```diff
@@ -57,6 +57,8 @@
       let xrViewerSpace = null;
       let xrHitTestSource = null;
 
+      const sessionInit = {requiredFeatures: ['local', 'hit-test']};
```

Comment:
'local' should always implicitly be a required feature for immersive sessions. If this was necessary to get the page to run properly that's a bug.

Reply:
It worked fine w/o it but I believe that explicit > implicit. Happy to remove it if you feel strongly about it, I'm not sure what the conventions are.